byteparse/jvm/src/test/scala/byteparse/LargeBmpIteratorTests.scala

```diff
@@ -0,0 +1,60 @@
+package byteparse
+
+import java.net.URL
+
+import fastparse.{IteratorParserInput, LoggedParsedInput}
+
+import utest._
+import fastparse.allByte._
+import BmpParser._
+
+object LargeBmpIteratorTests extends TestSuite {
+  val url = "https://raw.githubusercontent.com/lihaoyi/fastparse/master/byteparse/jvm/src/test/resources/lena.bmp"
+  def stream = new Iterator[IndexedSeq[Byte]] {
```

Comment:
I can write `Iterator[WrappedArray[Byte]]` which, I think, would be faster than simple `IndexedSeq`, but a little bit slower that `Array`.